python/hsfs/feature_store.py

```diff
@@ -415,6 +416,11 @@ def create_on_demand_feature_group(
             description: A string describing the contents of the on-demand feature group to
                 improve discoverability for Data Scientists, defaults to empty string
                 `""`.
+            primary_key: A list of feature names to be used as primary key for the
+                feature group. This primary key can be a composite key of multiple
+                features and will be used as joining key, if not specified otherwise.
+                Defaults to empty list `[]`, and the first column of the DataFrame will
+                be used as primary key.
```

Comment:
The documentation here is wrong. If you don't provide any primary key, then the feature group won't have any primary key at all. 

I believe you have taken the documentation from the `create_feature_group` method. The documentation is wrong there as well. Can you please fix it in both places?